src/textual/_doc.py

```diff
@@ -115,7 +119,11 @@ async def auto_pilot(pilot: Pilot) -> None:
             result = run_before(pilot)
             if inspect.isawaitable(result):
                 await result
+        await pilot.pause()
         await pilot.press(*press)
+        if hover:
+            await pilot.hover(hover)
+            await pilot.pause(0.5)
```

Comment:
For the purposes of testing and auto-pilots, isn't there something else we can call to trigger the tooltip immediately? Waiting half a second for a tooltip to show up in CI can compound to a lot of time.

Reply:
It is an issue, but anything else wouldn't be a fair test.

I did start implementing a virtual clock of sorts, that could run at 10x speed when testing. But that didn't work out.